content/posts/2021-features.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "2021 for Packit"
+date: 2022-01-12T09:23:38+01:00
+---
```

Comment:
```suggestion
date: 2022-01-12T09:23:38+01:00
draft: false
weight: 65
---
```

Reply:
Thanks.

Do we have some rules for `weight` or just pick a random number in the interval of neighbouring posts?

Reply:
I described the `weight` in #374 and also realized that we can actually really omit the `draft: false`.